packages/portal-proto/src/pages/_app.tsx

```diff
@@ -40,14 +40,13 @@ datadogRum.init({
   applicationId: "3faf9c0a-311f-4935-a596-3347666ef35d",
   clientToken: "pub9f7e31eaacd4afa71ac5161cbd5b0c11",
   site: "datadoghq.com",
-  service: "portal-2.0",
+  service: "portal2",
 
   // Specify a version number to identify the deployed version of your application in Datadog
   // version: '1.0.0',
   sessionSampleRate: 100,
   sessionReplaySampleRate: 0,
   trackUserInteractions: true,
-  trackFrustrations: true,
```

Comment:
Why remove this?

Reply:
With the upgrade from v4 to v5 [Frustration signals are collected automatically](https://docs.datadoghq.com/real_user_monitoring/guide/browser-sdk-upgrade/#frustration-signals-are-collected-automatically).

Reply:
Is this meant to be added to the master? Its not the current code branch develop is, or do you mean to update the production code?